ls1intum/Artemis
# `Development`: Remove unused button on exam details page

### Checklist
#### General


#### Client

### Motivation and Context



Fixes #6325.

### Description


Removed an unused button on the exam management page.
Took the opportunity to update some icons.

### Steps for Testing

Prerequisites:
- 1 Instructor
- 1 Exam

1. Log in to Artemis
2. Navigate to course administration and go to the exam details page
3. See there is no button for 'Participant Scores'

### Review Progress






#### Code Review
#### Manual Tests

### Test Coverage



Unchanged.

### Screenshots



![Screenshot](https://user-images.githubusercontent.com/102237363/223492658-7246a2ab-4408-4c13-a735-831fae2a9b17.png)
